Rakefile

```diff
@@ -6,5 +6,5 @@ require File.expand_path("config/application", __dir__)
 
 Rails.application.load_tasks
 
-task(:default).clear
+Rake::Task[:default].clear
```

Comment:
Do we even need this line? I notice it was [added 7 years ago with no context](https://github.com/alphagov/transition/commit/038876f26a4efd595510367b5356b9811899bbaf) and I don't think other apps usually do this? 🤔

Reply:
We actually do this [in Content Publisher](https://github.com/alphagov/content-publisher/blob/master/Rakefile#L9). The reason is that gems like RSpec (unhelpfully) shove themselves into the `:default` rake task. Here, we're overriding that behaviour so we can:

- Clearly specify what should be run, in one place
- Control the order of the tasks that are run

Reply:
Ah yes, that rings a bell now. 👍